Remotion/Web/Development.WebTesting/ScreenshotCreation/Fluent/IFluentScreenshotElement`.cs

```diff
@@ -45,5 +45,11 @@ public interface IFluentScreenshotElement<T> : IFluentScreenshotElement
     /// </summary>
     [NotNull]
     ScreenshotTransformationCollection<T> Transformations { get; set; }
+
+    /// <summary>
+    /// Scrolls the target element into view. This method must only be used with <see cref="Coypu.ElementScope"/> targets.
+    /// </summary>
+    /// <exception cref="InvalidOperationException">The target is not an <see cref="Coypu.ElementScope"/> instance.</exception>
+    FluentScreenshotElement<T> ScrollIntoView ();
```

Comment:
The new API is big enough and unrelated enough to warrent a separate ticket + branch. Do you agree that extracting it makes sense?